File: LeetCode/Easy/Score of a String/3110.java
class Solution {
    public int scoreOfString(String s) {
        int output = 0;

        char[] arr = s.toCharArray();
        for (int i = 0; i < arr.length - 1; i++) {
            output += Math.abs(arr[i] - arr[i + 1]);
        }

        return output;
    }
}

File: LeetCode/Easy/Score of a String/3110.js
/**
 * @param {string} s
 * @return {number}
 */
var scoreOfString = function(s) {
    return [...s].reduce((acc, _, i) => 
            (i < s.length - 1) ? acc + Math.abs(s.charCodeAt(i) - s.charCodeAt(i + 1)) : acc, 0);
};
